apps/api/src/modules/users/users.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@nestjs/common';
+import { CreateUserInput } from './dto/create-user.input';
+import { User } from './entities/user.entity';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { encodePassword } from 'src/common/utils/bcrypt';
+
+@Injectable()
+export class UsersService {
+  constructor(
+    @InjectRepository(User)
+    private readonly userRepository: Repository<User>,
+  ) {}
+
+  async createUser(createUserInput: CreateUserInput): Promise<User> {
```

Comment:
Can remove this for now due to the same reasons stated above. Don't need this yet.

Reply:
Ok, will remove